android/app/src/main/java/app/organicmaps/util/SharingUtils.java

```diff
@@ -142,10 +143,11 @@ public static void shareMapObject(@NonNull Context context, @NonNull MapObject o
 
     final String geoUrl = Framework.nativeGetGe0Url(object.getLat(), object.getLon(),
                                                     object.getScale(), object.getName());
+    final String coordUrl = String.format("https://omaps.app/%.5f,%.5f", object.getLat(), object.getLon());
```

Comment:
The idea is that the full link (that includes zoom, name, and maybe in the future some other parameters) is generated in C++. Then iOS, Android and desktop platforms can reuse it and extend it without unnecessary copy-pasting.